readthedocs/templates/builds/build_detail.html

```diff
@@ -53,6 +53,14 @@
           {{ build.builder }}
         </li>
       {% endif %}
+
+      <div data-bind="visible: finished()">
+        <li>
+          <a href="#">
+            {% trans "Raw log" %}
```

Comment:
"Raw log" --> "View raw"?

Reply:
Is the `log` word implicit here? Or what about `View raw log`?